api/src/public/login/style.css

```diff
@@ -147,6 +147,38 @@ form {
   text-decoration: none;
 }
 
+#password-container {
+  position: relative;
+  height: 40px;
+}
+
+#password-container #password {
+  width: 245px;
+  padding: 6px 43px 6px 10px;
+}
```

Comment:
Why is this necessary? If I delete it the password field looks the same to me...

Reply:
So letters don't go behind the icons
![Screenshot 2023-06-15 at 9 06 16 am](https://github.com/medic/cht-core/assets/66472237/598ed162-e7de-4306-93c2-5d091c2fff4d)

Reply:
Ahhh good thinking!